front/src/applications/stdcm/components/StdcmForm/StdcmConsist.tsx

```diff
@@ -116,6 +123,28 @@ const StdcmConsist = ({ disabled = false }: StdcmConfigCardProps) => {
     }
   }, [rollingStock]);
 
+  useEffect(() => {
+    const totalMassError = validateTotalMass(
+      rollingStock?.mass,
+      towedRollingStock?.mass,
+      totalMass
+    );
+
+    const totalLengthError = validateTotalLength(
+      rollingStock?.length,
+      towedRollingStock?.length,
+      totalLength
+    );
+
+    dispatch(
+      updateConsistErrors({
+        totalMass: totalMassError,
+        totalLength: totalLengthError,
+        maxSpeed: validateMaxSpeed(maxSpeed),
+      })
+    );
```

Comment:
Do we really need to use the store for errors? Could we instead `useMemo()` and pass the errors object as a prop?

`StdcmConsist` is a direct child of `StdcmConfig`.